errors.go

```diff
@@ -173,98 +173,99 @@ type KError int16
+	ErrSASLAuthenticationFailed            KError = 58 // Errors.SASL_AUTHENTICATION_FAILED
+	ErrUnknownProducerID                   KError = 59 // Errors.UNKNOWN_PRODUCER_ID
+	ErrReassignmentInProgress              KError = 60 // Errors.REASSIGNMENT_IN_PROGRESS
+	ErrDelegationTokenAuthDisabled         KError = 61 // Errors.DELEGATION_TOKEN_AUTH_DISABLED
+	ErrDelegationTokenNotFound             KError = 62 // Errors.DELEGATION_TOKEN_NOT_FOUND
+	ErrDelegationTokenOwnerMismatch        KError = 63 // Errors.DELEGATION_TOKEN_OWNER_MISMATCH
+	ErrDelegationTokenRequestNotAllowed    KError = 64 // Errors.DELEGATION_TOKEN_REQUEST_NOT_ALLOWED
+	ErrDelegationTokenAuthorizationFailed  KError = 65 // Errors.DELEGATION_TOKEN_AUTHORIZATION_FAILED
+	ErrDelegationTokenExpired              KError = 66 // Errors.DELEGATION_TOKEN_EXPIRED
+	ErrInvalidPrincipalType                KError = 67 // Errors.INVALID_PRINCIPAL_TYPE
+	ErrNonEmptyGroup                       KError = 68 // Errors.NON_EMPTY_GROUP
+	ErrGroupIDNotFound                     KError = 69 // Errors.GROUP_ID_NOT_FOUND
+	ErrFetchSessionIDNotFound              KError = 70 // Errors.FETCH_SESSION_ID_NOT_FOUND
+	ErrInvalidFetchSessionEpoch            KError = 71 // Errors.INVALID_FETCH_SESSION_EPOCH
+	ErrListenerNotFound                    KError = 72 // Errors.LISTENER_NOT_FOUND
+	ErrTopicDeletionDisabled               KError = 73 // Errors.TOPIC_DELETION_DISABLED
+	ErrFencedLeaderEpoch                   KError = 74 // Errors.FENCED_LEADER_EPOCH
+	ErrUnknownLeaderEpoch                  KError = 75 // Errors.UNKNOWN_LEADER_EPOCH
+	ErrUnsupportedCompressionType          KError = 76 // Errors.UNSUPPORTED_COMPRESSION_TYPE
+	ErrStaleBrokerEpoch                    KError = 77 // Errors.STALE_BROKER_EPOCH
+	ErrOffsetNotAvailable                  KError = 78 // Errors.OFFSET_NOT_AVAILABLE
+	ErrMemberIdRequired                    KError = 79 // Errors.MEMBER_ID_REQUIRED
+	ErrPreferredLeaderNotAvailable         KError = 80 // Errors.PREFERRED_LEADER_NOT_AVAILABLE
+	ErrGroupMaxSizeReached                 KError = 81 // Errors.GROUP_MAX_SIZE_REACHED
+	ErrFencedInstancedId                   KError = 82 // Errors.FENCED_INSTANCE_ID
+	ErrEligibleLeadersNotAvailable         KError = 83 // Errors.ELIGIBLE_LEADERS_NOT_AVAILABLE
+	ErrElectionNotNeeded                   KError = 84 // Errors.ELECTION_NOT_NEEDED
+	ErrNoReassignmentInProgress            KError = 85 // Errors.NO_REASSIGNMENT_IN_PROGRESS
+	ErrGroupSubscribedToTopic              KError = 86 // Errors.GROUP_SUBSCRIBED_TO_TOPIC
+	ErrInvalidRecord                       KError = 87 // Errors.INVALID_RECORD
+	ErrUnstableOffsetCommit                KError = 88 // Errors.UNSTABLE_OFFSET_COMMIT
+	ErrThrottlingQuotaExceeded             KError = 89 // Errors.THROTTLING_QUOTA_EXCEEDED
+	ErrProducerFenced                      KError = 90 // Errors.PRODUCER_FENCED
+	ErrDescribeUserScramCredentialNotExist KError = 91 // User not founded with describe users
```

Comment:
https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/ZkAdminManager.scala#L805

The error does not seem to be attached to “Scram” specifically. 

The code is itself just “resource not found” https://fossies.org/dox/kafka-3.7.1-src/enumorg_1_1apache_1_1kafka_1_1common_1_1protocol_1_1Errors.html#a0232a0a06a6b03b6868b74862c6393e3

Reply:
Yes, code named by "resource not found", but used this error only with users. So, i can rename this in next commit?

Reply:
Whatever this name is when merged becomes the name that it has to be to keep from now until we can make a breaking change.

So, ideally, we want this error to be descriptive but also generic enough to cover all of its usage. And breaking with the common format and concurrence with `Errors.RESOURCE_NOT_FOUND` would violate the principle least astonishment.